scripts/ThermalExpansion.zs

```diff
@@ -84,7 +84,7 @@ for slab in <ore:slabWood>.items {
 	100000000, 1000000, 
 	<thermalexpansion:frame:132>, 
 	[<gendustry:genetics_processor>, <extrautils2:ingredients:5>, <ic2:plate:16>,
-	<ic2:lapotron_crystal>.anyDamage(), <forestry:thermionic_tubes:12>, <forestry:crafting_material:1>,
+	<ic2:lapotron_crystal:*>, <forestry:thermionic_tubes:12>, <forestry:crafting_material:1>,
```

Comment:
DOES THIS WORK!? Could've sworn I tried that :P

Reply:
It's what's used for Quantum armor, and also why they didn't break :D